sdks/python/apache_beam/examples/inference/README.md

```diff
@@ -0,0 +1,114 @@
+<!--
+    Licensed to the Apache Software Foundation (ASF) under one
+    or more contributor license agreements.  See the NOTICE file
+    distributed with this work for additional information
+    regarding copyright ownership.  The ASF licenses this file
+    to you under the Apache License, Version 2.0 (the
+    "License"); you may not use this file except in compliance
+    with the License.  You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing,
+    software distributed under the License is distributed on an
+    "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+    KIND, either express or implied.  See the License for the
+    specific language governing permissions and limitations
+    under the License.
+-->
+
+# Example RunInference API Pipelines
+
+This module contains example pipelines that use the Beam RunInference
+API. <!---TODO: Add link to full documentation on Beam website when it's published.-->
+
+## Pre-requisites
+
+You must have `apache-beam>=2.40.0` installed in order to run these pipelines,
+because the `apache_beam.examples.inference` module was added in that release.
+Using the RunInference API also `torch` to be installed. 
+
+To install for a local pipeline, run:
+```
+pip install apache-beam torch==1.11.0
+```
+
+To install for a Dataflow pipeline, refer to these
+[instructions](https://beam.apache.org/documentation/sdks/python-pipeline-dependencies/#pypi-dependencies).
+You'll need to add `torch` to a `requirements.txt` file, and then run your
+pipeline with the following command-line option:
+```
+--requirements_file requirements.txt
+```
+
+<!---
+TODO: Add link to full documentation on Beam website when it's published.
+
+i.e. "See the
+[documentation](https://beam.apache.org/documentation/dsls/dataframes/overview/#pre-requisites)
+for details."
+-->
+
+## Image Classification with ImageNet dataset
+
+[`pytorch_image_classification.py`](./pytorch_image_classification.py) contains
+an implementation for a RunInference pipeline thatpeforms image classification
+on [ImageNet dataset](https://www.image-net.org/) using the MobileNetV2
+architecture.
+
+The pipeline reads the images, performs basic preprocessing, passes them to the
+PyTorch implementation of RunInference, and then writes the predictions
+to a text file in GCS.
+
+### Data
+Data related to RunInference has been staged in
+`gs://apache-beam-ml/` for use with these example pipelines:
```

Comment:
I don't know gs://apache-beam-ml will really work as a link or is right.

maybe "staged in apache-beam-testing" will work

Feel free to keep it this way if I'm just wrong or misunderstanding.

Maybe this will link right if the users cloud account is set to apache-beam-testing.